composer.json

```diff
@@ -10,8 +10,11 @@
         "ext-hash": "*",
         "ext-json": "*",
         "psr/log": "~1.0",
+        "symfony/config": "^4.4 || ^5.0",
```

Comment:
should be in require-dev on root package

Reply:
Thank you